File: C/Iniciante/1150 - Ultrapassando Z..c
/*
* Autor: Pierre Vieira
* Data de submissão: 16/02/2019 21:08:21
*/
#include <stdio.h>
int main(){
    int x, z, cont = 1, soma;
    scanf("%d", &x);
    scanf("%d", &z);
    while (x >= z){
        scanf("%d", &z);
    }
    soma = x;
    while(soma < z){
        cont++;
        soma += x;
        x++;
    }
    printf("%d\n",cont);
    return 0;
}

File: C#/Iniciante/1150 - Ultrapassando Z.cs
/*
* Autor: Pierre Vieira
* Data de submissão: 17/05/2020 00:19:09
*/
using System;

namespace URI {
    class Program {
        static void Main(string[] args) {
            int x, z, cont = 1;
            x = int.Parse(Console.ReadLine());
            while (true) {
                z = int.Parse(Console.ReadLine());
                if (z > x) {
                    break;
                }
            }
            while (x <= z) {
                x += x + 1;
                cont++;
            }
            Console.WriteLine(cont+1);
        }
    }
}
